### ISSUE TITLE
error connecting: to device no such file or directory

### ISSUE THREAD

**Issue by kloytofyexploiter** [Ingan121/FSUntether#10]
how do i fix this iproxy 1338 1338
Creating listening port 1338 for device port 1338
waiting for connection
New connection for 1338->1338, fd = 5
waiting for connection
Requesting connecion to USB device handle 1 (serial: ), port 1338
Error connecting to device: No such file or directory

**Comment by kloytofyexploiter** [comment]
it happens when i run nc localhost 1338

**Comment by Ingan121** [comment]
Does `nc localhost 1338` work on-device using [a-Shell](https://apps.apple.com/kr/app/a-shell/id1473805438) or [LibTerm](https://apps.apple.com/us/app/libterm/id1380911705)?

**Comment by kloytofyexploiter** [comment]
No it doesn't when I run that command on my phone while proxy is trying to run nothing happens it's just waiting for connection

**Comment by Ingan121** [comment]
It seems something went wrong while building. Could you show me the build log output?
Also, what method did you use for installing FSUntether.ipa?

**Comment by kloytofyexploiter** [comment]
yeah maybe I built it wrong too because I know another guy had problems with the choosing options but to sign it I used scarlet since it use a dev certificate

**Comment by BomberFish** [comment]
This happens with me as well. Signed using an enterprise certificate. macOS 14.0 `23A5276g`, iOS 16.1.2 (AFU), iproxy 2.0.2-`24-g36ffb7a`.

Logs:
```bash
❯ iproxy 1338 1338
Creating listening port 1338 for device port 1338
waiting for connection
New connection for 1338->1338, fd = 5
waiting for connection
Requesting connecion to USB device handle 9 (serial: 00008020-00XXXXXXXXXXXXXX), port 1338
Error connecting to device: No such file or directory
```
